src/main/java/org/toradocu/translator/ConditionTranslator.java

```diff
@@ -443,6 +444,8 @@ private static void processTag(Tag tag, DocumentedMethod method) {
                     "Shouldn't be",
                     ". " + ((ParamTag) tag).parameter().getName() + " shouldn't be");
       }
+
+      if (tag.getKind() == Tag.Kind.RETURN) {}
```

Comment:
What's the purpose of this if block? As a placeholder for the @return translation code?